README.md

```diff
@@ -1,8 +1,33 @@
+# navigation-next
 
 ## Introduction
-The navigation-next repo contains a complete Search, Mapping and Navigation platform provided by Comtech Telecommuncations to the Genivi forum. The source code provided is design for an Intel x86 architecture, using the Intel NUC as the demo hardware. ARM based platforms are also available. Comtech has been providing navigation systems for mobile and automotive platforms for over 10 years. The navigation-next platform supports map data in Comtech’s NCDB format, this format supports map data from TomTom, Here along with Comtech’s Trusted OSM maps. Trusted OSM maps are certified to be automotive grade and suitable for navigation and search. The project contains a complete OpenGL based map rendering engine, routing engine along with a search engine for searching a POI database.
+
+The navigation-next repo contains a complete Search, Mapping and
+Navigation platform provided by Comtech Telecommuncations to the
```

Comment:
A spelling error (carried over from the original):
Telecommuncations -> Telecommunications